src/Controls/src/Core/IElementController.cs

```diff
@@ -11,7 +11,7 @@ public interface IElementController
 
 		void SetValueFromRenderer(BindableProperty property, object value);
 		void SetValueFromRenderer(BindablePropertyKey propertyKey, object value);
-		ReadOnlyCollection<Element> LogicalChildren { get; }
+		IReadOnlyList<Element> LogicalChildren { get; }
```

Comment:
`ReadOnlyCollection` was both `IReadOnlyCollection` and `IReadOnlyList`, and we are using the indexer in places.